plugin.xml

```diff
@@ -26,6 +39,12 @@
         </config-file>
         <config-file parent="/*" target="AndroidManifest.xml">
         </config-file>
+        
```

Comment:
The "param" above should be something like:
```
<param name="android-package" value="com.urbanairship.accengage-cordova.UAirshipPlugin" />
```
Take a look at `plugin.xml` in the `urbanairship-cordova` repo for how our main Cordova package does it.

Reply:
ok :)